supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -101,13 +101,27 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use the following text verbatim, where `<tech_preview_name>` is the name of your product, component or feature:
+
+// The following is based on RHEL 9 snip_techpreview.adoc
 ----
-In <release>, a Technology Preview is available for <feature explanation>.
+<tech_preview_name> is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
 ----
+
+// PREVIOUS GUIDANCE
+// In <release>, a Technology Preview is available for <feature explanation>.
+
+.Examples of possible values for <tech_preview_name>
+
+* The Driver Toolkit
+* SSPI connection support on Microsoft Windows
+* Hot-plugging virtual disks
+* The Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants
+
 .Example doc text
+
 ----
-In Red Hat OpenStack Platform 16.0, a Technology Preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants.
+The Driver Toolkit is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
```

Comment:
Hey @stoobie this all looks like great changes here, just a thought: would it be possible to change the anthropomorphic 'enabling' (because, why not if possible) but no worries if it needs to be in line with legal wording etc.

Reply:
Hey @ndeevy,

Thanks for reviewing!

I took this text verbatim as noted in #81.

But this is the first time I've ever heard any issue with the word "enabling". How is it anthropomorphic? What do you use instead?

Reply:
Ok cool, I guess it's legal team approved then since it's from that link in #81  Apologies for the noise, we can leave it as is, there might be legal reasoning behind the words here. 

For the sake of discussing the mechanics of language, style, and grammar though ... :-D  certain words jump out that might indicate anthropomorphism. Like allow, let, permit, and enable. If the idea is that an object, technology, or non-human entity inherits human abilities or gives permission to a human to do something, it's probably anthropomorphic and puts the human in a subordinate position below the object. Usually this can be reworked to put the human in front so it's user-oriented, (sometimes not though because RH technologies do sometimes inherit human abilities).

In this case, I'd probably change it to something like:

`These features provide customers early access to upcoming product features to test functionality and provide feedback during the development process.`

There's an entry in the _IBM Style Guide: quick reference_ on confluence too if you want to look there -any feedback very welcome.

Reply:
Hmmm...I do see what you mean. I never really liked "enable" so I try to avoid it, but it's nice to see it mentioned in the style guide. Thanks for pointing that out!
This particular text is not something that I'm going to change. It's been hammered out already.